_search-plugins/knn/knn-vector-quantization.md

```diff
@@ -13,6 +13,10 @@ By default, the k-NN plugin supports the indexing and querying of vectors of typ
 
 OpenSearch supports many varieties of quantization. In general, the level of quantization will provide a trade-off between the accuracy of the nearest neighbor search and the size of the memory footprint consumed by the vector search. The supported types include byte vectors, 16-bit scalar quantization, and product quantization (PQ).
 
+## Faiss byte vector
```

Comment:
Let's just update Lucene byte vector as Byte vector and update the content there.